jenkinsfiles/docs.Jenkinsfile

```diff
@@ -44,7 +44,7 @@ pipeline {
             steps {
                 Status("PENDING", "${env.JOB_NAME}")
                 checkout scm
-                sh "make test-docs"
+                sh "make -C Documentation html"
```

Comment:
Should we actually keep this job or disable it to reduce load Jenkins (if nothing else)?

Reply:
+1 we need it in the CI mostly to detect misspelled words